src/components/subPages/Notes/Notes.component.jsx

```diff
@@ -8,26 +8,90 @@ import HeadingTwo from "../../UI/Headings/HeadingTwo/HeadingTwo.component";
 import HeadingThree from "../../UI/Headings/HeadingThree/HeadingThree.component";
 import HeadingFour from "../../UI/Headings/HeadingFour/HeadingFour.component";
 import ReportContainer from "../../UI/ReportContainer/ReportContainer.component";
+import ReportField from "../../UI/ReportField/ReportField.component";
 
 // subPage: Notes
-function Notes() {
+function Notes({selectedPatient, notesData}) {
+  //#region presentingComplaintRender = Presenting Complaint report #TODO - Needs testing
+  const presentingComplaintRender = notesData.map (
+    ({id, AddtItem_PC}) => (
+      <NotesRender key={id}>
+        <NotesGrid>
+          <NotesColumn>
+            <ReportField
+              data={AddtItem_PC ? AddtItem_PC : "Not recorded"}
+            />
+          </NotesColumn>
+        </NotesGrid>
+      </NotesRender>
+    )
+  );
+  //#end region /presentingComplaintRender = Presenting Complaint report
```

Comment:
Just be careful with your commented regions - `//#end region` should be `//#endregion`